encoding/json/outbound.go

```diff
@@ -52,6 +59,9 @@ type Request struct {
 	// Request body. This may be any type that can be serialized by
 	// json.Marshal.
 	Body interface{}
+
+	// Ttlms is the ttl in ms
+	Ttlms int
```

Comment:
Sure we can figure out the naming later. I would fix the `Duration` thing regardless though. `Duration` makes it more difficult to confuse the unit being used and makes it unnecessary to include the unit name in the variable name.